scripts/docker/hub/publish-docker.sh

```diff
@@ -45,7 +32,8 @@ case "${SCHEDULE_TAG:-${CI_COMMIT_REF_NAME}}" in
             --tag "parity/parity:stable" \
             --file tools/Dockerfile .;
         docker push "parity/parity:${VERSION}-${CI_COMMIT_REF_NAME}";
-        docker push "parity/parity:stable";;
+        docker push "parity/parity:stable";
+        docker push "parity/parity:latest";;
```

Comment:
`latest` == `nightly`? Can we be consistent here and use the same term?

Reply:
I believe this is mainly to sticker to Docker's convention of using a `latest` tag, but I could be wrong

Reply:
`latest` is a by-default for docker and many use it this way. We could add `nightly` to be a second tag for `latest` if you will.

Reply:
No it's fine, if "latest" is the convention then we should use that. :)